WeatherPortal/weatherapp/views.py

```diff
@@ -22,12 +64,66 @@ def register(request):
             return redirect(reverse('weather_in_user_city'))
 
 
-def your_weather(request):
+def your_weather(request, add_new_location=False, chosen_location=0):
+    """
+        Connect with openweather api and show information about weather in choosen city in website
+        Saved added city for only logged user
+        Check validator that city exist and for logged user that city exist in user's saved cities
+    """
+    cities_locations = []
+    if request.method == 'POST':
+        form = CityForm(request.POST)
+        if form.is_valid():
+            new_add_city = form.cleaned_data['name']
+            get_location_for_new_city = get_city_location(new_add_city)
+            if len(get_location_for_new_city) == 1:
+                add_new_location = True
```

Comment:
To jest tymczasowe? Dalej widzę, że jeśli warunek będzie spełniony to coś się wyprintuje i nic więcej